packages/frontend-shared/src/components/ShikiHighlight.vue

```diff
@@ -64,7 +64,7 @@ shikiWrapperClasses computed property.
       v-if="copyButton"
       variant="outline"
       tabindex="-1"
-      class="bg-white absolute"
+      class="bg-white ml-auto -mt-[32px] sticky margin-top:"
```

Comment:
No need for brackets, can just do `-mt-32px`. Also, not sure what `margin-top:` is used for.

Reply:
Good catch, apparently I got ahead of myself with the copy-pasta